src/main/java/org/candlepin/resource/ProductResource.java

```diff
@@ -142,15 +156,21 @@ public ProductCertificateDTO getProductCertificate(String productUuid) {
             return null;
         }
 
-        ResultIterator<Owner> iterator = this.ownerCurator.getOwnersWithProducts(productUuids).iterate();
+        // PageRequest set to null to get all the records
+        PageRequest pageRequest = null;
```

Comment:
This is a sign that our abstraction is leaky, and why we should avoid the legacy paging stuff wherever possible.

Reply:
Removed.